Source/Shaders/BillboardCollectionVS.glsl

```diff
@@ -152,6 +166,14 @@ void main()
 
     vec2 imageSize = vec2(floor(temp), compressedAttribute2.w);
 
+#ifdef CLAMP_TO_GROUND
+    v_textureOffset = textureOffset;
+    v_depthLookupTextureCoordinate1 = (0.0, 0.9) - depthLookupST; //the origin
```

Comment:
Are you sure these are right? Say the anchor point is the bottom right, then wouldn't you want to test `vec2(-1.0, 1.0)` instead of `vec2(1.0)`?

Reply:
For the `vec*` constructors, a single argument duplicates it for all components.

Reply:
Yeah, you're right. I think I forgot to test different horizontal origins, this only works for center.